src/bundle/Resources/public/js/scripts/core/multilevel.popup.menu.js

```diff
@@ -0,0 +1,385 @@
+(function (global, doc, ibexa, Popper) {
+    class MultilevelPopupMenu {
+        constructor(config) {
```

Comment:
what about destructuring config?

Reply:
We tend to not destructure `config`s in our codebase so that it is more clear in the constructor which thing comes from it.